docs/cache-provider.md

```diff
@@ -0,0 +1,49 @@
+---
+title: 'CacheProvider'
+---
+
+It can be useful to customize emotion's options - i.e. to add custom Stylis plugins, customize prefix of inserted class names, render style tags into specific element and more. `createCache` accepts the very same options as [create-emotion](/packages/create-emotion#options) factory.
```

Comment:
I wouldn't mind showing the options here and linking to here from the create-emotion docs since the CacheProvider should be used more than create-emotion.

Reply:
Hm, sure - can reverse that relation. Gonna work on it some time later 👍

Reply:
I've tweaked the PR a little bit, didn't move Upsides/Downsides from create-emotion's docs, because I don't have strong intuition how to do it best.